zerver/openapi/zulip.yaml

```diff
@@ -6630,6 +6630,13 @@ paths:
                 type: array
                 items:
                   type: object
+                  properties:
+                    name:
+                      type: string
+                      description: The name of a stream.
+                    description:
+                      type: string
+                      description: The short description of the stream.
```

Comment:
Small thing here, but I would use 'A short description' vs 'The short description', because it's something that could be different depending on who was writing the description.

Also, after this section, I'd add a `required` attribute to the object because the `description` property will only be present when creating a new stream.

Reply:
> Small thing here, but I would use 'A short description' vs 'The short description', because it's something that could be different depending on who was writing the description.

Oh so is it like, for this POST request we know that description can be anything so we use "A", but other request in which we already know what the description is (for example a GET request that gets stream information), we will use "The", am I getting it right?

Reply:
Yes, exactly. And extending that example for a PATCH request, it would be something like 'A new description that replaces the current description.'

Reply:
Interesting🤔, thanks!